react/components/UserWidget.tsx

```diff
@@ -291,6 +293,24 @@ const UserWidget: VtexFunctionComponent<UserWidgetProps> = ({
       setLoadingState(false)
     }
 
+    const {
```

Comment:
I believe we only want to send this metric if the change team operation above succeeds, right? Or, if we do want to send it regardless of the result, so that we know the frequency that the operation was requested by users, maybe we should add a field indicating if it was successful or not? Not sure if that would be useful, just making sure the data is the one we need.